src/types/formatted/qaSection.ts

```diff
@@ -0,0 +1,12 @@
+import { FormattedParagraph } from '@/data/queries'
+import { PublishedParagraph } from './publishedEntity'
+import { HeadingLevel } from './headingElement'
+
+export type QaSection = PublishedParagraph & {
+  header: string
+  intro: string
+  displayAccordion: boolean
+  questions: FormattedParagraph[]
+  type: 'paragraph--q_a_section' | 'paragraph--q_a_group'
+  headerTagLevel?: HeadingLevel
```

Comment:
Looking again, this was meant to be refactored out as unnecessary as it isnt currently used in content-builds latest templates either. will clean up

Reply:
With that out of the way we largely have similar data that is being shaped similarly by formaters.  the group formatter is currently being used in resources and support, and I will be able to implement the section formatted version with vet-centers.  if it proves to be an issue there I can look to just create separated templates, but i think this could work